### PR TITLE
Updated training script for torch prescriptors to allow config

### PR DESCRIPTION
**Pull request by danyoungday**

### CODE REVIEW

File: use_cases/eluc/prescriptors/nsga2/configs/test.json
Comment (danyoungday) [inline use_cases/eluc/prescriptors/nsga2/configs/test.json:1]: Basic config file similar to the ESP one.

File: use_cases/eluc/prescriptors/nsga2/train_prescriptors.py [hunk 1]
```diff
@@ -1,32 +1,46 @@
...
 from predictors.neural_network.neural_net_predictor import NeuralNetPredictor
 
 if __name__ == "__main__":
+
+    # Load config
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config_path", type=str, required=True)
+    args = parser.parse_args()
+    with open(Path(args.config_path), "r", encoding="utf-8") as f:
+        config = json.load(f)
```
Comment (danyoungday) [inline use_cases/eluc/prescriptors/nsga2/train_prescriptors.py:22]: The only argument we need is a config file. More customizability can come in the subset of data we use, what type of predictor we use, etc. but for now this simple one works

File: use_cases/eluc/prescriptors/nsga2/train_prescriptors.py [hunk 2]
```diff
@@ -1,32 +1,46 @@
...
-    nnp.load("predictors/neural_network/trained_models/no_overlap_nn")
+    nnp_path = Path(config["predictor_path"])
+    nnp.load(nnp_path)
+
     print("Initializing prescription...")
-    candidate_params = {"in_size": len(constants.CAO_MAPPING["context"]),
-                        "hidden_size": 16,
-                        "out_size": len(constants.RECO_COLS)}
+    if "seed_dir" in config["evolution_params"].keys():
+        config["evolution_params"]["seed_dir"] = Path(config["evolution_params"]["seed_dir"])
```
Comment (danyoungday) [inline use_cases/eluc/prescriptors/nsga2/train_prescriptors.py:35]: Convert seed dir string into Path object here so that we don't have to do it inside the TorchPrescriptor object and can continue assuming that what gets passed in is a Path

**Review by ofrancon — APPROVED** [review]
